src/shared/codelens/localLambdaRunner.ts

```diff
@@ -172,11 +185,8 @@ export class LocalLambdaRunner {
         })
     }
 
-    /**
-     * Runs `sam local invoke` against the provided template file
-     * @param samTemplatePath sam template to run locally
-     */
-    private async invokeLambdaFunction(samTemplatePath: string): Promise<void> {
+    /** More indirection. */
+    private async run2(samTemplatePath: string, cfnResourceName: string): Promise<void> {
```

Comment:
the point is that this fun, and its containing class, are noise. They are deprecated. The old name was just distracting because it was identical to another unrelated function in the same file, which is the non-deprecated path.

Reply:
Removed this class entirely.